cartographer/.clang-format

```diff
@@ -0,0 +1 @@
+BasedOnStyle: Google
```

Comment:
Should we also add
```
DerivePointerAlignment: false
PointerAlignment: Left
```
Otherwise the first pointer alignment (int* a vs int *a) determines the formatting for the rest of the file?

Reply:
I can add this. Right now I just mirrored what wally does, i.e. set style to Google.

Reply:
Cool, please specify `PointerAlignment: Left` as we do in 95% of the code base

Reply:
I'll add `DerivePointerAlignment: false` which is the important part here. `Left` is already set in Google style by default. This is now similar to what tensorflow does. If you want me to explicitly set `PointerAlignment` just let me know.

Reply:
Makes sense, I think this should do what we expect it to.